automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/orch/OrchestrationApiSpec.scala

```diff
@@ -51,7 +50,10 @@ class OrchestrationApiSpec extends FreeSpec with Matchers with ScalaFutures with
 
         Orchestration.billing.addGoogleRoleToBillingProjectUser(projectName, user.email, role)(ownerToken)
 
-        val queryReference = googleBigQueryDAO.startQuery(userToken.value, GoogleProject(projectName), shakespeareQuery).futureValue
+        val queryReference = eventually {
```

Comment:
I'm not sure it makes sense to use both `eventually` and `.futureValue`? I think calling `.futureValue` already takes the `PatienceConfig` into account when polling for a result.

Reply:
hm ok - I'll try without

Reply:
facetalked - modified my approach, did not remove